Makefile

```diff
@@ -1,14 +1,14 @@
 # ====================================================================================
 # Setup Project
 
-PROJECT_NAME := provider-jet-template
+PROJECT_NAME := provider-tf-template
```

Comment:
This looks like a mistake?

Reply:
Yes for some reason my commit was outdated. I meant doing a one-liner change.